cmd/config/etcd/help.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * MinIO Cloud Storage, (C) 2019 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package etcd
+
+import "github.com/minio/minio/cmd/config"
+
+// etcd config documented in default config
+var (
+	Help = config.HelpKV{
+		Endpoints:      `(required) Comma separated list of etcd endpoints eg: "http://localhost:2379"`,
+		CoreDNSPath:    `(optional) CoreDNS etcd path location to populate DNS srv records eg: "/skydns"`,
```

Comment:
should CoreDNSPath be configurable here? If the config is changed from /skydns to another dns path /abcd, will that not cause any entries created in etcd prior to the change to fail to be discovered? Verified that this is indeed the case.
```
MINIO_ETCD_ENDPOINTS=http://localhost:2379
MINIO_DOMAIN=example.com
./minio server ~/export
➜  mc git:(fbf61127) mc admin config get myminio etcd
# This is a default etcd configuration
etcd client_cert="" client_cert_key="" coredns_path="/skyxxx" endpoints=",,,," state="on" 

➜  mc git:(fbf61127) mc mb myminio/testbucket
Bucket created successfully `myminio/testbucket`.
➜  mc git:(fbf61127) mc admin config set myminio etcd coredns_path="/skyban" state="on" endpoints=",,,,"
etcd coredns_path=/skyban state=on endpoints=,,,,
Setting new key has been successful.
Please restart your server with `mc admin service restart myminio`.

➜  mc git:(fbf61127) mc admin service restart myminio
Restart command successfully sent to `myminio`.
Restarted `myminio` successfully.
➜  mc git:(fbf61127) mc ls myminio
➜  mc git:(fbf61127) mc mb myminio/newbucket
Bucket created successfully `myminio/newbucket`.
➜  mc git:(fbf61127) mc ls myminio          
[2019-10-30 00:07:34 PDT]      0B newbucket/

➜  mc git:(fbf61127) etcdctl get --from-key ""
/skyban/com/example/newbucket/10.0.0.224
{"host":"10.0.0.224","port":"9000","ttl":30,"creationDate":"2019-10-30T07:07:34.78378272Z"}
/skyban/com/example/newbucket/172.17.0.1
{"host":"172.17.0.1","port":"9000","ttl":30,"creationDate":"2019-10-30T07:07:34.786005807Z"}
/skyban/com/example/newbucket/172.21.0.1
{"host":"172.21.0.1","port":"9000","ttl":30,"creationDate":"2019-10-30T07:07:34.783342665Z"}
/skyban/com/example/newbucket/192.168.122.1
{"host":"192.168.122.1","port":"9000","ttl":30,"creationDate":"2019-10-30T07:07:34.787920173Z"}
/skyxxx/com/example/testbucket/10.0.0.224
{"host":"10.0.0.224","port":"9000","ttl":30,"creationDate":"2019-10-30T07:06:26.749494938Z"}
/skyxxx/com/example/testbucket/172.17.0.1
{"host":"10.0.0.224","port":"9000","ttl":30,"creationDate":"2019-10-30T07:06:26.749494938Z"}
/skyxxx/com/example/testbucket/172.17.0.1
{"host":"172.17.0.1","port":"9000","ttl":30,"creationDate":"2019-10-30T07:06:26.75272093Z"}
/skyxxx/com/example/testbucket/172.21.0.1
{"host":"172.21.0.1","port":"9000","ttl":30,"creationDate":"2019-10-30T07:06:26.749931679Z"}
/skyxxx/com/example/testbucket/192.168.122.1

```

Reply:
> should CoreDNSPath be configurable here? If the config is changed from /skydns to another dns path /abcd, will that not cause any entries created in etcd prior to the change to fail to be discovered? Verified that this is indeed the case.

Yes, but why would anyone keep changing this - they have to change this such that they are changing their CoreDNS Corefile as well to match the PATH @poornas - this is not an issue instead a feature. 

So any path change here will not migrate your existing entries, it is basically a one-time setting and that's it.  It is also done to avoid isolation for certain clusters to use their own custom path for DNS discovery with different CoreDNS deployments but single etcd.

Reply:
To understand more read here https://github.com/minio/minio/issues/8433